### ISSUE TITLE
Storage Meta Data

### ISSUE THREAD

**Issue by sfxworks** [distriqt/ANE-Firebase#89]
Does anyone happen to know of the proper way to get and compare a file's metadata?

https://stackoverflow.com/questions/45174642/firebase-md5-bloddycrytpos-dont-match?noredirect=1#comment77320815_45174642

**Comment by marchbold** [comment]
Can't say I've tried to match the hash as yet so I'm not sure how it's generated. Assume you've had a look through the Firebase storage documentation?

**Comment by sfxworks** [comment]
Yes, it's supposed to be a base64 of an md5 but I can't get them to match. Details in the comments of the thread I linked.

**Comment by marchbold** [comment]
Yeah I saw, probably a question for the Firebase team?

**Comment by marchbold** [comment]
Not sure if this is any help: https://stackoverflow.com/questions/41380078/firebase-storage-getmetadata-issue

Demonstrates generating the hash in java

**Comment by sfxworks** [comment]
I'll take a look at that. I saw it in the thread too but didn't pay enough attention to it. From a skim it looks like it has an answer. If not I already made a post in the firebase quick examples for android github. I appreciate it.

**Comment by marchbold** [comment]
Let me know how you go

**Comment by sfxworks** [comment]
https://stackoverflow.com/questions/45174642/firebase-md5-bloddycrytpos-dont-match/45203627#45203627 Got it resolved! Looks like you have to parse the string in a different way than the prebuilt functions ByteArrays or Strings have. It would be cool if this was a small function in the build.

**Comment by marchbold** [comment]
Okay, will see if we can get this added into a utilities class.
